readme.md

```diff
@@ -1,57 +1,81 @@
-# DSPause
+<h1 align="center">
+ds-pause
+</h1>
 
-_`delegatecall` based proxy with an enforced wait time for code execution_
+<p align="center">
+<i><code>delegatecall</code> based proxy with an enforced delay</i>
+</p>
 
-`ds-pause` is designed to be used as a component in a governance system, to ensure that the governed
-have time to respond to malicious actions. This can help to constrain the power wielded by the
-governors.
+`ds-pause` is designed to be used as a component in a governance system, to give affected parties
+time to respond to decisions. If those affected by governance decisions have e.g. exit or veto
+rights, then the pause can serve as a effective check on governance power.
 
 ## Plans
 
-`ds-pause` allows authorized entities to make `plans`. A `plan` describes a single `delegatecall`
-operation and a unix timestamp `eta` before which it cannot be executed.
+A `plan` describes a single `delegatecall` operation and a unix timestamp `eta` before which it
+cannot be executed.
 
-Once the `eta` has passed, a `plan` can be executed by anyone.
+A `plan` consists of:
 
-A `plan` can only be made if its `eta` is after `block.timestamp + delay`. The `delay` is
-configurable upon construction.
+- `usr`: address to `delegatecall` into
+- `fax`: `calldata` to use
+- `eta`: first possible (unix) time of execution
 
-A `plan` consists of:
+Each plan has a unique id, defined as `keccack256(abi.encode(usr, fax, eta))`
+
+## Operations
+
+Plans can be manipulated in the following ways:
+
+- **`plot`**: mark a `plan` as executable
+- **`exec`**: execute a `plan`
+- **`drop`**: cancel a `plan`
+
+## Invariants
 
-- `usr`: the address to `delegatecall` into
-- `fax`: the `calldata` to use
-- `eta`: the time from when the `plan` can be executed
+**`plot`**
+- A `plan` can only be plotted if it's `eta` is after `block.timestamp + delay`
```

Comment:
"its" 😅

Reply:
What would I do without you :pray: :grinning: